website/app/styles/pages/application/cover.scss

```diff
@@ -1,31 +1,69 @@
 // COVER
 
 @use "../../breakpoints" as breakpoint;
+@use "../../typography/mixins" as *;
 
 .doc-page-cover {
   position: relative;
   grid-area: cover;
-  // TODO check with heather what she wants to do with the vertical padding (responsive)
-  padding: 16px var(--doc-page-stage-gutter-small);
-  background-color: #fff;
-  border-bottom: 1px solid #dbdbdc;
+  min-width: 0; // This is needed to allow the scrolling of the tabs
+  padding: 16px var(--doc-page-stage-gutter-small) 0;
+  background-color: var(--doc-color-white);
+  border-bottom: 1px solid var(--doc-color-gray-500);
 
   @include breakpoint.medium () {
-    padding: 32px var(--doc-page-stage-gutter-medium);
+    padding: 64px var(--doc-page-stage-gutter-medium) 68px;
   }
 
   @include breakpoint.large () {
-    padding: 64px var(--doc-page-stage-gutter-large);
+    padding: 64px var(--doc-page-stage-gutter-large) 68px;
   }
 }
 
+.doc-page-cover__title-extra-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 24px;
+  align-items: center;
+}
+
 .doc-page-cover__title {
   margin: 0;
-  color: #1d1e1f; // TODO convert to token
+  color: var(--doc-color-gray-100);
+}
+
+
+.doc-page-cover__extra {
+  display: flex;
+  gap: 12px;
+}
+
+.doc-page-cover__link {
+  @include doc-font-style-body-small();
+  display: flex;
+  gap: 4px;
+  align-items: center;
 }
 
 .doc-page-cover__description {
   max-width: var(--doc-page-content-max-width);
   margin: 16px 0 0 0;
-  color: #727274; // TODO convert to token
+  color: var(--doc-color-gray-300);
+}
+
+.doc-page-cover__tabs {
+  display: flex;
+  flex-wrap: nowrap;
+  height: min-content;
+  margin-top: 24px;
+  overflow-x: auto;
+  overflow-y: visible; // needed or it will show a small vertical scrollbar
+  -webkit-overflow-scrolling: touch;
+
+  @include breakpoint.medium-and-above () {
+    position: absolute;
```

Comment:
I usually try to avoid using absolute positioning for something like this. It seems like maybe you could remove the bottom padding of .doc-page-cover instead (plus maybe adjusting the margin)?

Reply:
@KristinLBradley I though about it, and even started to make the changes, but then realized that since some pages have tabs and others don't, then I have to do different calculations and pass around props to account the different cases, so I decided to roll back the changes. we can revisit this later, when we will have more stable content and we can see what are the emerging patterns.